# joint information of a Trajectory
# moveit produce a array of joint angle information of a Trajectory
# name : name of the group of robot
# array of joint angle with rad angle
# clockwise for positive angle rotation
# example: 
#   name: "bear_controller"
#   Joint_1: [0.0, 0.00010871130966166027, 0.00021742261932332054, 0.0003261339289849808, 0.0004348452386466411]
#   Joint_2: [0.0, 0.12978129163444832, 0.25956258326889664, 0.389343874903345, 0.5191251665377933]
#   Joint_3: [0.0, -0.18046657043188732, -0.36093314086377465, -0.541399711295662, -0.7218662817275493]
#   Joint_4: [0.0, 0.0002442117065784289, 0.0004884234131568578, 0.0007326351197352867, 0.0009768468263137155]
#   Joint_5: [0.0, -0.05047370984661126, -0.10094741969322252, -0.15142112953983378, -0.20189483938644504]
#   Joint_6: [0.0, -0.00016488822387519326, -0.0003297764477503865, -0.0004946646716255797, -0.000659552895500773]
#   time: [0.0, 0.60046657, 0.848436509, 1.0964064470000001, 1.696873018]
#   topic name: goal_pub



Header header

string name
float64[] Joint_1
float64[] Joint_2
float64[] Joint_3
float64[] Joint_4
float64[] Joint_5
float64[] Joint_6
float64[] time
